BoldGrid/crio
# Issue 48 - Add notice to Design > Footer > Colors to help users

ISSUE: Resolves #48 

PROJECT: [Crio 2.20.0](https://github.com/orgs/BoldGrid/projects/13/views/9)

# Add notice to Design > Footer > Colors to help users #

## Test Files ##

[crio-2.20.0-issue-48.zip](https://github.com/BoldGrid/crio/files/11092736/crio-2.20.0-issue-48.zip)

## NOTES TO TESTER ##
### Please leave comments regarding issues found in testing directly on the issue linked above, not the Pull Request. This helps ease the process of reviewing the testing of multiple PRs in a given project from the project view. ###
### Please remember to move this item from 'Needs Review' to either 'In Progress' or 'Awaiting Release' depending on the results of your testing ###

## Testing Instructions ##

1. Install the test zip files linked above.
2. Open the WP Customizer and navigate to `Design > Footer > Colors`
3. Verify that the Footer Links title has been renamed to `Attribution Link Colors`
4. Open the `Related Options` tab, and review the content of the help section
5. Click the links in the help info paragraph, as well as in the `Link Color Elements` section of the dropdown. Make sure they direct you to the associated panels inside the customizer.
6. Click the FAQs links, and ensure they open new tabs to the correct support articles.
